ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/compact-3node-cluster/provision/cucushift-installer-rehearse-aws-ipi-compact-3node-cluster-provision-chain.yaml

```diff
@@ -0,0 +1,12 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-compact-3node-cluster-provision
```

Comment:
It's better to follow the same naming convention as Azure: `cucushift-installer-rehearse-azure-ipi-compact-provision`

Reply:
My parent directory name is set to this, because in order to unify the name with OCPQE-13612, and avoid the cluster with other number of nodes in the future.